docs/docs-components/siteIndex.js

```diff
@@ -42,11 +42,11 @@ const siteIndex: $ReadOnlyArray<siteIndexType> = [
   },
   {
     sectionName: 'Android',
-    pages: ['Overview', 'Avatar', 'IconButton', 'Tabs'],
+    pages: ['Overview'],
   },
   {
     sectionName: 'iOS',
-    pages: ['Overview', 'Avatar', 'IconButton', 'Tabs'],
+    pages: ['Overview', 'Avatar', 'IconButton', 'Tabs', 'Button', 'Sheet'],
```

Comment:
We should probably alphabetize the components.

Reply:
will do